scipy/stats/tests/test_qmc.py

```diff
@@ -445,63 +457,50 @@ class TestHalton(QMCEngineTests):
 class TestLHS(QMCEngineTests):
     qmce = qmc.LatinHypercube
     can_scramble = False
-    unscramble_nd = np.array([[0.73412877, 0.50416027],
-                              [0.5924405, 0.51284543],
-                              [0.57790629, 0.70797228],
-                              [0.44357794, 0.64496811],
-                              [0.23461223, 0.55712172],
-                              [0.45337347, 0.4440004],
-                              [0.73381992, 0.01751516],
-                              [0.52245145, 0.33099331]])
+
+    scramble = [True, False]
+    ids = ["Scrambled", "Unscrambled"]
```

Comment:
Already defined in the base class
```suggestion
```

Reply:
That is what I thought but it causes the error below...

===================================================================================================== test session starts ======================================================================================================
platform linux -- Python 3.8.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/kei/Desktop/scipy, configfile: pytest.ini
collected 5482 items / 1 error / 386 deselected / 5095 selected                                                                                                                                                                

============================================================================================================ ERRORS ============================================================================================================
________________________________________________________________________________________ ERROR collecting scipy/stats/tests/test_qmc.py ________________________________________________________________________________________
../../scipy/stats/tests/test_qmc.py:457: in <module>
    class TestLHS(QMCEngineTests):
        Counter    = <class 'collections.Counter'>
        QMCEngine  = <class 'scipy.stats._qmc.QMCEngine'>
        QMCEngineTests = <class 'scipy.stats.tests.test_qmc.QMCEngineTests'>
        RandomEngine = <class 'scipy.stats.tests.test_qmc.RandomEngine'>
        TestHalton = <class 'scipy.stats.tests.test_qmc.TestHalton'>
        TestUtils  = <class 'scipy.stats.tests.test_qmc.TestUtils'>
        TestVDC    = <class 'scipy.stats.tests.test_qmc.TestVDC'>
        __builtins__ = <builtins>
        __cached__ = '/home/kei/Desktop/scipy/scipy/stats/tests/__pycache__/test_qmc.cpython-38.pyc'
        __doc__    = None
        __file__   = '/home/kei/Desktop/scipy/scipy/stats/tests/test_qmc.py'
        __loader__ = <_pytest.assertion.rewrite.AssertionRewritingHook object at 0x7fdfa2c261c0>
        __name__   = 'scipy.stats.tests.test_qmc'
        __package__ = 'scipy.stats.tests'
        __spec__   = ModuleSpec(name='scipy.stats.tests.test_qmc', loader=<_pytest.assertion.rewrite.AssertionRewritingHook object at 0x7fdfa2c261c0>, origin='/home/kei/Desktop/scipy/scipy/stats/tests/test_qmc.py')
        _test_find_index = <built-in function _test_find_index>
        assert_    = <function assert_ at 0x7fdf95eaa160>
        assert_allclose = <function assert_allclose at 0x7fdf95eaaf70>
        assert_almost_equal = <function assert_almost_equal at 0x7fdf95eaa670>
        assert_array_almost_equal = <function assert_array_almost_equal at 0x7fdf95eaa8b0>
        assert_array_equal = <function assert_array_equal at 0x7fdf95eaa820>
        assert_equal = <function assert_equal at 0x7fdf95eaa550>
        check_random_state = <function check_random_state at 0x7fdf84d75a60>
        n_primes   = <function n_primes at 0x7fdf84d75d30>
        np         = <module 'numpy' from '/home/kei/Desktop/dev_env_scipy/lib/python3.8/site-packages/numpy/__init__.py'>
        primes_from_2_to = <function primes_from_2_to at 0x7fdf84d75ca0>
        pytest     = <module 'pytest' from '/home/kei/Desktop/dev_env_scipy/lib/python3.8/site-packages/pytest/__init__.py'>
        qmc        = <module 'scipy.stats.qmc' from '/home/kei/Desktop/scipy/scipy/stats/qmc.py'>
        shapiro    = <function shapiro at 0x7fdf84d5a3a0>
        test_subclassing_QMCEngine = <function test_subclassing_QMCEngine at 0x7fdf8096d5e0>
        update_discrepancy = <function update_discrepancy at 0x7fdf84d75c10>
        van_der_corput = <function van_der_corput at 0x7fdf84d75dc0>
../../scipy/stats/tests/test_qmc.py:494: in TestLHS
    ???
E   NameError: name 'scramble' is not defined
        __module__ = 'scipy.stats.tests.test_qmc'
        __qualname__ = 'TestLHS'
        can_scramble = False
        qmce       = <class 'scipy.stats._qmc.LatinHypercube'>
        test_continuing = <function TestLHS.test_continuing at 0x7fdf80902670>
        test_fast_forward = <function TestLHS.test_fast_forward at 0x7fdf80902550>
        test_sample = <function TestLHS.test_sample at 0x7fdf809025e0>
        test_sample_stratified = <function TestLHS.test_sample_stratified at 0x7fdf809024c0>
=================================================================================================== short test summary info ====================================================================================================
ERROR ../../scipy/stats/tests/test_qmc.py - NameError: name 'scramble' is not defined
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================================================================================== 386 deselected, 1 error in 1.23s ===============================================================================================

Reply:
Ah, I missed the next comment.